app/lib/screens/study/dashboard/settings.dart

```diff
@@ -93,53 +95,145 @@ class _SettingsState extends State<Settings> {
         child: Column(
-          mainAxisSize: MainAxisSize.min,
-          mainAxisAlignment: MainAxisAlignment.spaceAround,
-          children: <Widget>[
-            getDropdownRow(context),
-            const SizedBox(height: 24),
-            Text(
-              '${AppLocalizations.of(context)!.study_current} ${subject!.study.title}',
-              style: theme.textTheme.titleLarge,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(
+              children: [
+                const Icon(Icons.fast_forward, color: Colors.orange, size: 20),
+                const SizedBox(width: 8),
+                Text(
+                  'Debug Study Mode',
+                  style: Theme.of(context).textTheme.titleMedium?.copyWith(
+                    color: Colors.orange,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ],
+            ),
+            const SizedBox(height: 16),
+            Row(
+              children: [
+                const Text('Enable Next Day button: '),
+                Switch(
+                  value: appState.isDebugModeEnabled,
+                  onChanged: (value) async {
+                    await appState.setDebugModeEnabled(value);
+                  },
+                ),
+              ],
             ),
             const SizedBox(height: 8),
-            ElevatedButton.icon(
-              icon: Icon(MdiIcons.exitToApp),
-              label: Text(AppLocalizations.of(context)!.opt_out),
-              style: ElevatedButton.styleFrom(
-                backgroundColor: Colors.orange[800],
+            Text(
+              'When enabled, a "Next Day" button will appear on the dashboard to fast-forward the study.',
+              style: Theme.of(context).textTheme.bodySmall?.copyWith(
+                color: Colors.grey[600],
+                fontStyle: FontStyle.italic,
               ),
-              onPressed: () {
-                showDialog(
-                  context: context,
-                  builder: (_) => OptOutAlertDialog(subject: subject),
-                );
-              },
             ),
-            const SizedBox(height: 24),
-            ElevatedButton.icon(
-              icon: const Icon(Icons.delete),
-              label: Text(AppLocalizations.of(context)!.delete_data),
-              style: ElevatedButton.styleFrom(backgroundColor: Colors.red),
-              onPressed: () {
-                showDialog(
-                  context: context,
-                  builder: (_) => DeleteAlertDialog(subject: subject),
-                );
-              },
+            const SizedBox(height: 8),
+            Text(
+              'Tap the settings title 7 times to show/hide these controls',
+              style: Theme.of(context).textTheme.bodySmall?.copyWith(
+                color: Colors.grey[600],
+                fontStyle: FontStyle.italic,
+              ),
```

Comment:
We already have a hidden debug menu, so we can just add a switch there. No need for another hiding place